ui/webui/test/check-storage

```diff
@@ -422,6 +423,7 @@ class TestStorageMountPoints(anacondalib.VirtInstallMachineCase, StorageHelpers)
         applied_partitioning = s.dbus_get_applied_partitioning()
 
         # When adding a new partition a new partitioning should be created
+        i.back()
```

Comment:
This will now work for cases where the accounts screen is disabled (eg Workstation). We may need something like reach() for going back.